bears/general/SpaceConsistencyBear.py

```diff
@@ -38,7 +41,47 @@ def run(self,
         result_texts = []
         additional_info_texts = []
 
-        for line_number, line in enumerate(file, start=1):
+        def get_blanklines_nr_start():
+            line_nr_start = False
+            rev_enumerated_zip_obj = zip(reversed(range(1, len(file) + 1)),
+                                         reversed(file))
+            for line_number, line in rev_enumerated_zip_obj:
+                replacement = line
+                if replacement.strip() == '':
+                    line_nr_start = line_number
+                else:
+                    break
+
+            return line_nr_start
+
+        if allow_trailing_blanklines:
+            end_line_of_file = len(file)
+
+        else:
+            blanklines_nr_start = get_blanklines_nr_start()
+            end_line_of_file = len(file)
+            if blanklines_nr_start:
```

Comment:
Instead of letting the `get_blanklines_nr_start` function return `None` if no blanklines found, rather compare it against `len(file)`, makes things simpler :+1: